package.json

```diff
@@ -4,9 +4,11 @@
   "description": "A handsome dashboard framework written in javascript",
   "main": "app.js",
   "scripts": {
-    "start": "node app.js",
-    "postinstall": "webpack -p --progress --colors",
+    "start": "babel-node app.js",
+    "postinstall": "npm run build",
```

Comment:
What's the difference between 'build' and 'postinstall' here?

Do we still want to be calling npm?